### PR TITLE
Adds a few admin helper procs (spawn character, despawn character)

### PR DESCRIPTION
**Pull request by PolymorphBlue**
Also makes follow a valid  verb again.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added follow verb, some admin helper verbs.
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -505,7 +505,53 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
...
+	set category = "Ghost"
+	set name = "Follow" // "Haunt"
+	set desc = "Follow and haunt a mob."
+
+	if(!input)
+		input = input(usr, "Select a mob:", "Ghost Follow") as null|anything in getmobs_ghost_follow()
+	if(!input)
+		return
+
+	var/target = getmobs_ghost_follow()[input]
```
Comment (out-of-phaze) [inline code/modules/mob/dead/observer/observer.dm:550]: Is this necessary?  `getmobs_ghost_follow` seems kind of expensive, and this means it's called three times:
- when fetching the arguments for the verb
- when running the verb without an argument
- when getting `target` from the name of the mob.
It's *probably* fine as-is, but ideally we'd have it be argument-less and cache the result of `getmobs_ghost_follow()` instead.

**Review by out-of-phaze — CHANGES_REQUESTED** [review]
Just a couple things I noticed. Mostly good to go, though.

File: code/modules/mob/dead/observer/observer.dm (hunk 1 above)
Comment (PolymorphBlue) [reply code/modules/mob/dead/observer/observer.dm:550]: Maybe not;  I snagged all three of these from citcode while kinda head empty no thoughts.

I'll go though the logic and consider it.

Also, thank you for the review!

Comment (PolymorphBlue) [reply code/modules/mob/dead/observer/observer.dm:550]: Considering it a bit more, I think the fact it's a user-triggered verb (meaning calls are relatively rare) makes it an acceptable tradeoff.  Having getmobs_ghost_follow in the argument list is required for autocompletion in the text bar, and I do like that and use it.  I don't think there's a good way to optimize it while keeping that behavior.

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**